packages/legacy/core/App/screens/Splash.tsx

```diff
@@ -210,6 +212,10 @@ const Splash: React.FC = () => {
       try {
         const credentials = await getWalletCredentials()
 
+        if (typeof ocaBundleResolver['checkForUpdates'] != "undefined") {
+          await ocaBundleResolver.checkForUpdates()
+        }
```

Comment:
Would it be cleaner to change this to just `await ocaBundleResolver.checkForUpdates?.()` (no if statement)

Reply:
Updated without the if statement

Reply:
Looks good! You may need to rebase your branch, it seems to be out of date with the base branch.

Reply:
Ah, and it looks like you'll need to run the lint step and fix any issues that arise. `yarn lint` from the root